requirements.txt

```diff
@@ -2,4 +2,6 @@ Django==4.1.2
 mysqlclient==2.1.1
 selenium
 coverage==4.5.2
-locust
\ No newline at end of file
+locust
+pandas==1.5.2
```

Comment:
The 'Pandas' requirement should be removed, as it is not used.

Reply:
You are right, I've just deleted it.